templates/taxi/driver_detail.html

```diff
@@ -3,6 +3,13 @@
 {% block content %}
   <h1>
     Username: {{ driver.username }}
+    <a href="{% url 'taxi:driver-delete' pk=driver.id %}" class="btn btn-danger link-to-page">
+      Delete
+    </a>
+
+    <a href="{% url 'taxi:driver-update' pk=driver.id %}" class="btn btn-secondary link-to-page">
+      Update
```

Comment:
is it anything in particular button "update" does? 
or you update driver as a whole?